index.html

```diff
@@ -2,29 +2,86 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./css/main.css" />
-    <link rel="icon" href="./favicon.ico" />
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="IE=edge"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link
+      rel="stylesheet"
+      href="./css/main.css"
+    />
+    <link
+      rel="icon"
+      href="./favicon.ico"
+    />
     <title>Aziz Mejri - Portfolio</title>
   </head>
+
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button
+        type="button"
+        class="menu-bars"
+        id="menu_btn"
+      >
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
-            <a class="nav-link" href="#contact"
+            <a
+              class="nav-link"
+              href="#projects"
+              >Portoflio</a
+            >
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#about-me"
+              >About</a
+            >
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#contact"
               >Contact <i class="far fa-envelope"></i
             ></a>
           </li>
         </ul>
       </nav>
+      <nav id="mobile-nav">
+        <span>x</span>
+        <ul>
+          <li>
+            <a
+              class="nav-link"
+              href="#"
+              >Portfolio</a
+            >
+          </li>
+          <li>
+            <a
+              class="nav-link"
+              href="#aboutme"
+              >About</a
+            >
```

Comment:
- [ ]  I notice a missing link to navigate to the portfolio section please you need to fix it 😄
- [ ]  For the "About" link, href="#aboutme" is correct assuming there is an element on the page with the id "aboutme" that the link should navigate to.
 Please make sure to verify that the element with the id "aboutme" exists in your HTML and adjust the href values according to your intended functionality.

Reply:
Good job in implementing this requested change 🥰🥰